lib/teamsnap.rb

```diff
@@ -1,3 +1,10 @@
+require "teamsnap/version"
+require "teamsnap/error"
+require "teamsnap/configuration"
+require "teamsnap/client"
+require "teamsnap/collection"
+require "teamsnap/item"
+
```

Comment:
I see you've given up on the idea of one file for the entire project. The only reason Kyle and I did this originally was to try and make the project so simple you only needed one file. Also, you didn't have to jump between files to understand everything. :smile: 

That being said, I yield to the wisdom of the group.